src/main/java/org/dwcj/interfaces/ICssClassable.java

```diff
@@ -0,0 +1,18 @@
+package org.dwcj.interfaces;
+
+public interface ICssClassable {
```

Comment:
What about the following for a name ?

- 'HasClassName'
- 'SupportsClassName'